bundles/ruby/ruby_mode.moon

```diff
@@ -68,7 +68,7 @@ continuation_indent = (line, indent_level) ->
   code_blocks:
     multiline: {
       { r'\\s+do(?:\\s*\\|[^|]+\\|)?\\s*$', '^%s*end', 'end' },
-      { r'^\\s*def\\s+\\w[\\w.\\d]+(?:\\s*\\([^)]*\\))?\\s*$', '^%s*end', 'end' },
+      { r'^\\s*def\\s+\\w[\\w.\\d?!]+(?:\\s*\\([^)]*\\))?\\s*$', '^%s*end', 'end' },
```

Comment:
The ruby bundle has a spec for the indentation support. This should be updated to cover these cases (you can run the specs with `./bin/howl-spec bundles/ruby/spec/`).